pkg/backup/backup_manager.go

```diff
@@ -79,6 +92,25 @@ func (bm *BackupManager) SaveSnap(lastSnapRev int64) (*backupapi.BackupStatus, e
 	return bs, nil
 }
 
+// SaveSnapWithPath saves latest snapshot to a given path.
+func (bm *BackupManager) SaveSnapWithPath(path string) error {
```

Comment:
Where is this used?

Reply:
I am think that this should be used in backup operator to save snapshot to a specific path.

Reply:
Can you submit the changes for backup operator using it?

Reply:
do you want me to include in this pr?

Reply:
Yes, please

Reply:
@hongchaodeng should we also change backup spec to take in a path too?

Reply:
Not right now.